buildsupport.sh

```diff
@@ -0,0 +1,8 @@
+mkdir dist/img
+mkdir dist/cache
+mkdir dist/fonts
```

Comment:
Please do not add `dist/fonts` as this causes a build error. I already removed it as part of https://github.com/LibertyGlobal/refapp/commit/bb86c8c870f6714e727cff08b6a8a9327969f413